### ISSUE TITLE
Unable to use 'import.meta.url'

### ISSUE THREAD

**Issue by PassTheMayo** [eslint/eslint#11544]
**Tell us about your environment**

* **ESLint Version:** 5.15.3
* **Node Version:** 11.12.0
* **npm Version:** 6.7.0

**What parser (default, Babel-ESLint, etc.) are you using?** Default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
	"env": {
		"browser": true,
		"es6": true,
		"node": true
	},
	"extends": "eslint:recommended",
	"globals": {
		"Atomics": "readonly",
		"SharedArrayBuffer": "readonly"
	},
	"parserOptions": {
		"ecmaVersion": 2018,
		"sourceType": "module"
	},
	"rules": {
		"indent": [
			"error",
			"tab"
		],
		"linebreak-style": [
			"error",
			"windows"
		],
		"quotes": [
			"error",
			"single"
		],
		"semi": [
			"error",
			"always"
		]
	}
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const __dirname = path.dirname(new URL(import.meta.url).pathname).substr(1);
```

```bash
eslint src/Website.mjs
```

**What did you expect to happen?** I didn't expect an error to be on that line, as it's valid.

**What actually happened? Please include the actual, raw output from ESLint.**
`8:40  error  Parsing error: Unexpected token import`

**Are you willing to submit a pull request to fix this bug?** Probably not

**Comment by ilyavolodin** [comment]
`import.meta` is a stage 3 proposal currently. ESLint only supports stage 4 proposals. For everything else, you can use `babel-eslint`.

**Comment by platinumazure** [comment]
Closing as the question has been answered. Please feel free to stop by our [Gitter chat](https://gitter.im/eslint/eslint) with future questions.

**Comment by dandv** [comment]
Duplicate of #11189 and #11803.
